addons/website_slides/static/lib/pdfslidesviewer/PDFSlidesViewer.js

```diff
@@ -148,6 +148,23 @@ var PDFSlidesViewer = (function(){
     PDFSlidesViewer.prototype.toggleFullScreen = function(){
         var el = this.canvas;
 
+        var $PDFViewer = $("div#PDFSlideViewer")
+        var $navigation_footer = $('div#PDFViewer > div.navbar-fixed-bottom')
```

Comment:
Isn't it a specific selector for the footer ? If so, use it instead of `div#PDFViewer > div.navbar-fixed-bottom`